website/layouts/releases/single.html

```diff
@@ -32,6 +32,29 @@
       </div>
       {{ end }}
 
+      <div class="mt-8">
```

Comment:
Borrowed this from the shortcode for warnings (`./website/layouts/shortcodes/warning.html`).